configs/config.go

```diff
@@ -89,22 +91,23 @@ func GetUserConfig(configFile string) (Config, error) {
 	return userCfg, nil
 }
 
-func GetIconForSeverity(severity FindingSeverityType, severities []SeverityConfig) (string, error) {
+func GetIconForSeverity(severity FindingSeverityType, severities []SeverityConfig) string {
 	for _, config := range severities {
 		if config.Label == SeverityNames[severity] {
-			return config.Slack_emoji, nil
+			return config.Slack_emoji
 		}
 	}
-	return "", fmt.Errorf("No Slack icon available for severity %s", SeverityNames[severity])
+	return DEFAULT_SLACK_ICON
+
 }
```

Comment:
Instead of returning an error if we don't find an icon, i think it's better if we fallback to the default icon. Should simplify the codebase.